src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -819,25 +883,34 @@ class misc_commandscript : public CommandScript
                                      handler->GetSession()->GetPlayer()->GetPositionZ(),
                                      handler->GetSession()->GetPlayer()->GetOrientation(),
                                      handler->GetSession()->GetPlayer()->GetZoneId(),
-                                     targetGuid);
+                                     target->GetGUID());
         }
 
         return true;
     }
+
     // Summon group of player
-    static bool HandleGroupSummonCommand(ChatHandler* handler, char const* args)
+    static bool HandleGroupSummonCommand(ChatHandler* handler, Optional<PlayerIdentifier> target)
     {
-        Player* target;
-        if (!handler->extractPlayerTarget((char*)args, &target))
+        if (!target)
+        {
+            target = PlayerIdentifier::FromTargetOrSelf(handler);
+        }
+
+        if (!target || !target->IsConnected())
+        {
             return false;
+        }
 
         // check online security
-        if (handler->HasLowerSecurity(target))
+        if (handler->HasLowerSecurity(target->GetConnectedPlayer()))
             return false;
 
-        Group* group = target->GetGroup();
+        auto targetPlayer = target->GetConnectedPlayer();
```

Comment:
Maybe change this part to (GitHub doesn't allow me to make a suggestion :(): 
```cpp
        auto targetPlayer = target->GetConnectedPlayer();

        // check online security
        if (handler->HasLowerSecurity(targetPlayer))
            return false;
```

Reply:
Yes, good idea